test/integration/connect/envoy/run-tests.sh

```diff
@@ -324,7 +326,7 @@ function suite_setup {
     docker run -d --name envoy_workdir_1 \
         $WORKDIR_SNIPPET \
         --net=none \
-        google/pause &>/dev/null
+        "${HASHICORP_DOCKER_PROXY}/google/pause" &>/dev/null
```

Comment:
We might consider temporarily switching to `k8s.gcr.io/pause:latest` until the cache is primed.